### ISSUE TITLE
Create button can be confusing "in context" (nit?)

### ISSUE THREAD

**Issue by colemickens** [kubernetes/dashboard#1220]
##### Environment

```
Dashboard version: 3230343cf811574bd369bd058f6d9c331843e69a
Kubernetes version: 1.4.0-beta something
Operating system: n/a
Node.js version: n/a
Go version: n/a
```
##### Steps to reproduce

Go to a Pod page. Notice that "edit" and "delete" are actions specific to the type. But it just always brings up the generic "Deploy app" dialog.

Not really a huge issue, and I guess there is the divider line now that kind of separates the contextual actions from the 'global' create.

Maybe it should say Deploy instead of Create?

**Comment by bryk** [comment]
Thanks @colemickens for the feedback! We're very happy that you're reporting this :) 

@romlein What do you think?

**Comment by romlein** [comment]
@colemickens agreed, we should standardize the _Deploy_ / _Create_ nomenclature. Thanks for reporting!
Though it's not an excuse for any UX issues we're observing now, as far as the _Create_ being a global object, that will be resolved by our [next phase](https://invis.io/CG8CVE87Z) addition of a top bar above the app bar.

Mockups for this version do have local actions as partially transparent so they're of a lower visual priority than "Create", but I don't know if we were thinking that would just be when they're disabled, or if it's incorrectly implemented.
cc @bryk?

**Comment by maciaszczykm** [comment]
This should be no problem anymore since we have two toolbars at the top. White one is for global actions and create is there:

![UI](https://raw.githubusercontent.com/kubernetes/dashboard/master/docs/dashboard-ui.png)

Blue one contains context actions like scale, edit and delete related to current view.

**Comment by maciaszczykm** [comment]
Create action should redirect to object creator wizard, but it is not yet implemented (#1600).
